lib/plugins/create/create.test.js

```diff
@@ -666,5 +666,36 @@ describe('Create', () => {
 
       return create.create().catch(() => download.downloadTemplateFromRepo.restore());
     });
+
+    it('should copy "aws-nodejs" template from local path', () => {
+      process.chdir(tmpDir);
+      const distDir = path.join(tmpDir, 'my-awesome-service');
+      create.options = {};
+      create.options.path = distDir;
+      create.options['template-path'] = path.join(__dirname, 'templates/aws-nodejs');
+
+      return create.create().then(() => {
+        const dirContent = fs.readdirSync(distDir);
+
+        expect(dirContent).to.include('serverless.yml');
+        expect(dirContent).to.include('handler.js');
+        expect(dirContent).to.include('gitignore');
+      });
+    });
+
+    it('should copy "aws-nodejs" template from local path with a custom name', () => {
```

Comment:
A custom name option makes a custom service name output to serverless.yml.
For example, running `sls create -t aws-nodejs -n test-service`, then `test-service` will be added to `service` section in serverless.yml.

However, it seems that this test case doesn't confirm that.